src/cryptoadvance/specter/server_endpoints/wallets_api.py

```diff
@@ -425,15 +427,26 @@ def addressinfo(wallet_alias):
         wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
         address = request.form.get("address", "")
         if address:
-            descriptor = wallet.get_descriptor(address=address)
+            descriptor = add_checksum(
```

Comment:
In the changed version we are generating two descriptors as strings at the api endpoint (and not as before in a confusing way with JS objects in the jinja template): one with pubkey and one with an xpub.